.github/scripts/cmd/cmd.py

```diff
@@ -137,8 +137,13 @@
             print(f'-- config: {config}')
             default_path = f"./{config['path']}/src/weights"
             xcm_path = f"./{config['path']}/src/weights/xcm"
-            output_path = default_path if not pallet.startswith("pallet_xcm_benchmarks") else xcm_path
-            print(f'-- benchmarking {pallet} in {runtime} into {output_path}')
+            template = None
+            if pallet.startswith("pallet_xcm_benchmarks"):
+                template = config['template']
```

Comment:
this condition if just for XCM weights, so I would be specific (aligned with matrix `benchmarks_xcm_template`):
```suggestion
                template = config['benchmarks_xcm_template']
```

maybe we should add to the `else` something like `template = config['template']` - but this is not necessary, we don't use template for others